docs/release-notes/cloud/cloud-2023-11-23-new-coverage-engine-status-checks.md

```diff
@@ -172,9 +170,25 @@ On December 7th 2023 we set the new Coverage engine to post coverage summaries t
     end="<!--accept-permission-end-->"
 %}
 
+### Quality dashboard, Coverage dashboard, Quality pull request list page, and Coverage pull request list page {: id="migrated-pages" }
```

Comment:
Actually wait, I am not sure if all the info is from new engine, I doubt it, I think we should not mention those

Reply:
Updated!